sources/packages/web/src/views/aest/institution/ProfileCreate.vue

```diff
@@ -36,17 +36,22 @@ export default defineComponent({
   setup() {
     const router = useRouter();
     const snackBar = useSnackBar();
+    const { excludeExtraneousValues } = useFormioUtils();
     const processing = ref(false);
     const institutionProfileModel = ref({
       clientType: ClientIdType.AEST,
       mode: "create",
     } as FormModel);
 
-    const createInstitution = async (data: AESTCreateInstitutionAPIInDTO) => {
+    const createInstitution = async (data: unknown) => {
       try {
```

Comment:
same as https://github.com/bcgov/SIMS/pull/2045/files#r1240372556

Reply:
Changed the liked comment to `unknow` and added a reply to it.